style.css

```diff
@@ -3,51 +3,62 @@ body {
   font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
 }
 
-header {
+.header-section {
   display: flex;
-  justify-content: space-around;
-  font-size: 1.7rem;
-  margin-bottom: 10px;
-  margin-right: 24px;
-  padding-top: 40px;
+  color: #fff;
   width: 100%;
-  height: 20px;
+  justify-content: space-between;
+  font-size: 18px;
 }
 
-.mylogo {
-  color: #fff;
-  font-style: normal;
-  font-weight: 500;
+.header-section h1 {
   font-size: 18px;
-  line-height: 16px;
-  flex: 1;
+  font-weight: 500;
+  line-height: 20px;
+  vertical-align: middle;
   padding-top: 10px;
-  margin-left: 20px;
-  text-decoration: none;
+  margin-left: 30px;
 }
 
-.mob-version {
+.nav-list {
+  display: none;
+}
+
+.workl {
+  display: none;
+}
+
+.mob-bar {
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 20px;
+  vertical-align: middle;
+  padding-top: 20px;
+  margin-right: 30px;
+}
+
+/* .mob-version {
   right: 50px;
   cursor: pointer;
   padding-right: 50px;
-}
+} */
```

Comment:
- [ ] Let's make our code clean by removing these commented lines of code.
- [ ] Consider adding padding to all the form input so that it gives it a little space around them.

Reply:
I have removed unused code from both files and added  padding to place holders of form input elements